docs/builder_image.md

```diff
@@ -177,4 +177,4 @@ should be the following:
 
 # Additional steps
 ## OpenShift support
-If you are intending to use this image with [OpenShift](https://github.com/openshift/origin), review and follow the OpenShift [image creation guidelines](https://docs.okd.io/latest/creating_images/guidelines.html).
+If you are intending to use this image with [OpenShift](https://github.com/openshift/origin), review and follow the OpenShift [image creation guidelines]([https://docs.okd.io/latest/creating_images/guidelines.html](https://docs.okd.io/latest/openshift_images/create-images.html)).
```

Comment:
i don't think this is the edit you intended to make

Reply:
Thanks for catching that, I used the `edit` feature on github, so maybe that messed it up and I pasted it.